src/types/submissionProblemScore.types.ts

```diff
@@ -0,0 +1,10 @@
+export type SubmissionProblemScore = {
+  id?: number
+  submissionId: number
+  assignmentProblemId: number
+  score: number
+  feedback: string
+  release?: string
```

Comment:
Should be released
```suggestion
  releasedAt?: string
```

Reply:
Since this will be used as a timestamp, can we use the convention for deleted/created/updatedAt for clarity